libraries/botbuilder-ai/src/qnaMakerDialog.ts

```diff
@@ -115,8 +115,18 @@ export interface QnAMakerDialogConfiguration extends DialogConfiguration {
  * It supports knowledge bases that include follow-up prompt and active learning features.
  * The dialog will also present user with appropriate multi-turn prompt or active learning options.
  */
-export class QnAMakerDialog extends WaterfallDialog implements QnAMakerDialogConfiguration {
```

Comment:
We want to enforce the interface, was this removed intentionally?

Reply:
whoops, definitely not. Conflict resolving fail 🤦